django/contrib/admindocs/views.py

```diff
@@ -206,7 +205,7 @@ def get_context_data(self, **kwargs):
         for field in opts.fields:
             # ForeignKey is a special case since the field will actually be a
             # descriptor that returns the other object
-            if isinstance(field, models.ForeignKey):
+            if field.many_to_one:
```

Comment:
True, it's good practice though cause at some point fields will contain all of the fields.